content/en/docs/Contribute/style-guide.md

```diff
@@ -0,0 +1,675 @@
+{{< /table >}}
+
+### Use code style for filenames, directories, and paths
+
+{{< table caption = "Do and Don't - Use code style for filenames, directories, and paths" >}}
+Do | Don't
+:--| :-----
+Open the `envars.yaml` file. | Open the envars.yaml file.
+Go to the `/docs/tutorials` directory. | Go to the /docs/tutorials directory.
+Open the `/_data/concepts.yaml` file. | Open the /\_data/concepts.yaml file.
+{{< /table >}}
+
+### Use the international standard for punctuation inside quotes
+
+{{< table caption = "Do and Don't - Use the international standard for punctuation inside quotes" >}}
+Do | Don't
+:--| :-----
+events are recorded with an associated "stage". | events are recorded with an associated "stage."
+The copy is called a "fork". | The copy is called a "fork."
+{{< /table >}}
+
+## Inline code formatting
+
+### Use code style for inline code, commands, and API objects {#code-style-inline-code}
+
+For inline code in an HTML document, use the `<code>` tag. In a Markdown
+document, use the backtick (`` ` ``).
+
+{{< table caption = "Do and Don't - Use code style for inline code, commands, and API objects" >}}
+Do | Don't
+:--| :-----
+The `kubectl run` command creates a `Pod`. | The "kubectl run" command creates a pod.
+The kubelet on each node acquires a `Lease`… | The kubelet on each node acquires a lease…
+A `PersistentVolume` represents durable storage… | A Persistent Volume represents durable storage…
+For declarative management, use `kubectl apply`. | For declarative management, use "kubectl apply".
+Enclose code samples with triple backticks. (\`\`\`)| Enclose code samples with any other syntax.
+Use single backticks to enclose inline code. For example, `var example = true`. | Use two asterisks (`**`) or an underscore (`_`) to enclose inline code. For example, **var example = true**.
+Use triple backticks before and after a multi-line block of code for fenced code blocks. | Use multi-line blocks of code to create diagrams, flowcharts, or other illustrations.
+Use meaningful variable names that have a context. | Use variable names such as 'foo','bar', and 'baz' that are not meaningful and lack context.
+Remove trailing spaces in the code. | Add trailing spaces in the code, where these are important, because the screen reader will read out the spaces as well.
+{{< /table >}}
+
+{{< note >}}
+The website supports syntax highlighting for code samples, but specifying a language
+is optional. Syntax highlighting in the code block should conform to the
+[contrast guidelines.](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=141%2C143#contrast-minimum)
+{{< /note >}}
+
+### Use code style for object field names and namespaces
+
+{{< table caption = "Do and Don't - Use code style for object field names" >}}
+Do | Don't
+:--| :-----
+Set the value of the `replicas` field in the configuration file. | Set the value of the "replicas" field in the configuration file.
```

Comment:
While not immediately necessary we should consider making the example Otel related later on.